docs/source/configuration.rst

```diff
@@ -261,37 +259,40 @@ Placeholder templating can be enabled in the config using:
 
 A few common styles are supported:
 
-colon
- WHERE bla = :my_name
+.. code-block:: sql
+   :force:
+
+    -- colon
+    WHERE bla = :my_name
```

Comment:
Decided to format this all as SQL (with a `:force:` since it's not SQL in places so breaks build otherwise).

Old:

![image](https://user-images.githubusercontent.com/10931297/150039238-9ed9590f-fd8c-486f-bc95-ea9ee7ca932f.png)


New:

![image](https://user-images.githubusercontent.com/10931297/150039252-46881d18-b732-48bc-bb1b-4700603d99da.png)

Reply:
Looks good 👍